### ISSUE TITLE
Pressing tab in bible verse highlights projector slide button

### ISSUE THREAD

**Issue by calebchanwy** [calebchanwy/tjc-powerpoint#5]
When pressing tab within the Bible verse selection, it should highlight the following in circular order:

1. Book
2. Chapter
3. Verse

However, the tab order is currently including the currently selected projector slide. This should be fixed so that whenever tab is pressed after the verse text box, it will focus back to the Book text box rather than going to the projector slide.
